vernac/mltop.mli

```diff
@@ -45,6 +45,8 @@ type toplevel =
 (** Sets and initializes a toplevel (if any) *)
 val set_top : toplevel -> unit
 
+val load_module : string -> unit
```

Comment:
Isn't that basically all mltop?

Reply:
Some plugins are not using coqpp, and in the past the language was `load_module`, so this could lead to errors that will be hard to spot.

I did a large cleanup not long ago but I'm not looking foward to do it again.

Reply:
So indeed the answer to your question I think is "for now we haven't declared Mltop as private"